challengeutils/__main__.py

```diff
@@ -461,6 +483,45 @@ def build_parser():
         help="Time quota submission has to run in milliseconds")
     parser_kill_docker.set_defaults(func=command_kill_docker_over_quota)
 
+    parser_set_quota = subparsers.add_parser(
+        'set-evaluation-quota',
+        help='Sets the quota on an existing evaluation queue')
```

Comment:
Maybe add that quota will reset if no optional arguments are passed? Since some people (myself included) might assume that nothing will happen if no flags are used.